spec/ruby/core/encoding/converter/last_error_spec.rb

```diff
@@ -55,29 +55,23 @@
     it "returns an Encoding::InvalidByteSequenceError when the last call to #convert produced one" do
       ec = Encoding::Converter.new("utf-8", "iso-8859-1")
       exception = nil
-      lambda do
-        begin
-          ec.convert("\xf1abcd")
-        rescue Encoding::InvalidByteSequenceError => e
-          exception = e
-          raise e
-        end
-      end.should raise_error(Encoding::InvalidByteSequenceError)
+      -> {
```

Comment:
Do these specs still need to be able to run on old Ruby implementations?

Reply:
From MRI 2.2.
But this doesn't change anything, it's just a better MSpec style.

Reply:
That's fine. I didn't know if stabby lambdas were allowed or not.

Reply:
It seems stabby lambdas are since 1.9 according to https://makandracards.com/makandra/17305-short-lambda-syntax-in-ruby-1-9